lib/resque/scheduler/scheduling_extensions.rb

```diff
@@ -149,6 +174,77 @@ def remove_schedule(name)
 
       private
 
+      def supports_lua?
```

Comment:
This and `redis_master_version` are already defined in Resque::Scheduler::Locking.  Why not just define it once and use it in both places.